src/components/Header/index.tsx

```diff
@@ -10,8 +10,22 @@ import ThemeToggler from './ThemeToggler'
 import { Logo } from './Logo'
 
 import Container from '../Container'
+import { HeaderQuery } from '../../types/generated'
+
+const Header = () => {
+  const { site } = useStaticQuery<HeaderQuery>(graphql`
```

Comment:
I replaced all `StaticQuery` with `useStaticQuery` to reuse query type (e.g. here `HeaderQuery`)

Reply:
oh genius! woooow, you didn't have to do that. so nice!